<?xml version="1.0"?>

<launch>
  <!-- face detection node (detects faces in color image and publishes their positions) -->
  <rosparam command="load" ns="/cob_people_detection/face_detector" file="$(find cob_people_detection)/ros/launch/face_detector_params.yaml"/>
  <node name="face_detector" pkg="cob_people_detection" ns="/cob_people_detection/face_detector" type="face_detector_node" output="screen">
    <remap from="head_positions" to="/cob_people_detection/head_detector/head_positions"/>
	
    <param name="data_directory" type="string" value="$(find cob_people_detection)/common/files/"/>
  </node>

</launch>
